lost/src/main/java/com/mapzen/android/lost/internal/ResolveLocationActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.mapzen.android.lost.internal;
+
+import android.app.Activity;
+import android.bluetooth.BluetoothAdapter;
+import android.content.Intent;
+import android.os.Bundle;
+import android.provider.Settings;
+
+/**
+ * Used to enable bluetooth before firing intent to open location settings activity.
+ */
+public class ResolveLocationActivity extends Activity {
+
+  private static final int REQUEST_CODE = 1;
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    BluetoothAdapter adapter = BluetoothAdapter.getDefaultAdapter();
```

Comment:
Per documentation `BluetoothAdapter.getDefaultAdapter()` can return null "if Bluetooth is not supported on this hardware platform". We should check for null before calling `adapter.enable()`.

https://developer.android.com/reference/android/bluetooth/BluetoothAdapter.html#getDefaultAdapter()